studio/local_queue.py

```diff
@@ -67,8 +67,8 @@ def delete(self):
                     pass
 
     def dequeue(self, acknowledge=True, timeout=0):
-        wait_step = 1
-        for waited in range(0, timeout + wait_step, wait_step):
+        sleep_in_seconds = 1
+        while True:
```

Comment:
Remove local queue own timeout - we rely on external shutdown action.
Overall shutdown latency will be approx. sleep_in_seconds sec.